libs/datamodel/core/src/dml/datamodel.rs

```diff
@@ -6,21 +6,13 @@ use super::*;
 /// string, only introspection and the lowering of the datamodel to the ast care about these flags.
 /// The FieldType: Unsupported behaves in the same way.
 /// Both of these are never converted into the internal datamodel.
-
-/// Represents a prisma-datamodel.
 #[derive(Debug, PartialEq, Clone)]
 pub struct Datamodel {
-    /// All enums.
     pub enums: Vec<Enum>,
-    /// All models.
     pub models: Vec<Model>,
 }
```

Comment:
Did you mean to delete these comments?

Reply:
Yes because they were not adding information compared to the names.